_layouts/default.html

```diff
@@ -141,8 +141,8 @@
             });
         </script>
     {% endif %}
-
-    {% include schema-event.html %}
+    
+    <!-- {% include schema-event.html %} -->
```

Comment:
We should add the add the schema here in the layouts instead of `head.html` since the head template contains only meta tags or link tags only.